core/src/main/java/io/grpc/package-info.java

```diff
@@ -32,40 +32,38 @@
 /**
  * The gRPC core public API.
  *
- * <p>gRPC is based on a client-server model of remote procedural calls.  A client creates a
- * channel which is connected to a server.  RPCs are initiated from the client and sent to the
- * server which then responds back to the client.  When the client and server are done sending
- * messages, they half close their respective connections.  The RPC is complete as soon as the
- * server closes.
+ * <p>gRPC is based on a client-server model of remote procedure calls. A client creates a channel
```

Comment:
Are there any fixes here? It only looks like debatable changes to line wrapping.